app/test_main.py

```diff
@@ -1,3 +1,13 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_the_animal_is_one_year_old() -> None:
+    assert get_human_age(23, 15) == [1, 1]
+
+
+def test_the_animal_is_two_year_old() -> None:
+    assert get_human_age(24, 27) == [2, 2]
+
+
+def test_each_subsequent_year() -> None:
+    assert get_human_age(28, 29) == [3, 3]
```

Comment:
Please read this file https://github.com/mate-academy/py-cat-and-dog-years/blob/master/checklist.md